Elevel/Elevel.Api/Controllers/AuditionController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Elevel.Application.Features.AuditionFeatures;
+using MediatR;
+
+namespace Elevel.Api.Controllers
+{
+
+    [Authorize, Route("[controller]/[action]")]
+    public class AuditionController : BaseApiController
+    {
+
+        [HttpPost]
+        public async Task<IActionResult> CreateAuditionAsync([FromBody] CreateAuditionCommand.Request request)
+        {
+            var res = await Mediator.Send(request);
+            return Ok(res);
+        }
+        [HttpPut("{id:Guid}")]
+        public async Task<IActionResult> UpdateAuditionAsync([FromRoute] Guid id, [FromBody] UpdateAuditionCommand.Request request)
+        {
+            if (id != request.Id)
```

Comment:
Why are you checking the same variable?
It is equal to 
if(true)

Reply:
it's not the same, it's just double check that we updating same object, not by mistake

Reply:
@parys I think it is not necessary I would be leave only the Request parameter.

Reply:
it's another holywar theme, but it can be, it's OK